docker-compose.mk

```diff
@@ -0,0 +1,46 @@
+MKFILE_PATH := $(abspath $(lastword $(MAKEFILE_LIST)))
+PROJECT_PATH := $(patsubst %/,%,$(dir $(MKFILE_PATH)))
+
+DOCKER_COMPOSE_VERSION := 1.21.0
+DOCKER_COMPOSE := $(BIN_PATH)/docker-compose
+DOCKER_COMPOSE_BIN := $(DOCKER_COMPOSE)-$(DOCKER_COMPOSE_VERSION)
+
+ifeq ($(CI),true)
+DOCKER_ENV = CI=true
+else
+DOCKER_ENV = CI=jenkins
```

Comment:
we don't need jenkins anymore 😄

Reply:
I won't fix `docker-compose.mk`, @Martouta. I'm putting it back so it doesn't break `openshift/system/Makefile`.

Reply:
@guicassolato I'm really not a big fan of introducing unnecessary code to master again after having merged https://github.com/3scale/porta/pull/1071 😞 , so may I fix it in another PR pointing to this one then if you don't want to do it yourself ? 🤔 If you disagree, I won't oppose strongly either 👍

Reply:
@Martouta, https://github.com/3scale/porta/pull/908 was incomplete and introduced a bug.  Don't bother much about `docker-compose.mk`. It's `openshift/system/Makefile` that needs to be fixed.

We can even remove https://github.com/3scale/porta/blob/1ffee4935d9ffb1a2bd025c540f7bca7141531a3/openshift/system/Makefile#L26 and delete `docker-compose.mk` again if we fix the targets that depend on it. They are probably broken already anyway.

Reply:
Ok then as @duduribeiro says, we need to create a Jira issue to completely fix all that for real since this PR is just a patch 😄

Reply:
https://issues.jboss.org/browse/THREESCALE-3246